### ISSUE TITLE
A11y_VSCode_ Integrated Terminal_UsabilityMAC: Unecessary decorative content announced for screen reader user.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#76878]
**Environment Details:**
VSCode Version : 1.36
macOS Mojave: v10.14.5
Mac mini (Late 2012) 

**Additional Details:**
Usability

**Repro Steps:**
1.Open Visual studio code editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3.Navigate to terminal action toolbar.
4.Navigate to combo box and expand it using alt+down arrow key.
5.Navigate to separator.

**Actual:**
When screen reader users navigate to the combo box and separator, it is announcing it as part of list.
Unnecessary decorative content is announced for screen reader user.

**Expected:**
Screen reader focus should not go on separator and it should not announce anything for separator.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader user will get confused when it announce separator as part of list.

**Attachment for Reference:**
[VSCode_Integrated Terminal_UsabilityMAC Unecessary decorative content announced for screen reader user.pptx](https://github.com/microsoft/vscode/files/3368463/VSCode_Integrated.Terminal_UsabilityMAC.Unecessary.decorative.content.announced.for.screen.reader.user.pptx)
